pkg/cmd/scan/output/output.go

```diff
@@ -64,6 +62,16 @@ func GetOutput(config OutputConfig, quiet bool) Output {
 	}
 }
 
+func ShouldPrint(outputs []OutputConfig, quiet bool) bool {
```

Comment:
It can be usefull to add context as comment for this function, maybe we can rename it to `isOutputAllowed` or something like that

Reply:
I added a comment, please tell me if it makes it clearer, I can also rename it.